src/HubBootstrapper.php

```diff
@@ -0,0 +1,180 @@
+        private TransportFactoryInterface $transportFactory,
+        private LoggerInterface $logger,
+        private HubInterface $hub,
+        private ContainerInterface $container,
+    ) {
+    }
+
+    public function bootstrap(): void
+    {
+        $this->options->setIntegrations(fn (array $integrations) => $this->prepareIntegrations($integrations));
+
+        $clientBuilder = new ClientBuilder($this->options);
+        $clientBuilder
+            ->setTransportFactory($this->transportFactory)
+            ->setLogger($this->logger);
+
+        $client = $clientBuilder->getClient();
+
+        $hub = $this->hub;
+        $hub->bindClient($client);
+        SentrySdk::setCurrentHub($hub);
+    }
+
+    /**
+     * @param IntegrationInterface[] $integrations
+     *
+     * @return IntegrationInterface[]
+     */
+    public function prepareIntegrations(array $integrations)
+    {
+        $userIntegrations = $this->resolveIntegrationsFromUserConfig();
+        if ($this->options->hasDefaultIntegrations()) {
+            $integrations = array_filter(
+                $integrations,
+                static function (
+                    SdkIntegration\IntegrationInterface $integration
+                ): bool {
+                    if (
+                        $integration instanceof
+                        SdkIntegration\ErrorListenerIntegration
+                    ) {
+                        return false;
+                    }
+
+                    if (
+                        $integration instanceof
+                        SdkIntegration\ExceptionListenerIntegration
+                    ) {
+                        return false;
+                    }
+
+                    if (
+                        $integration instanceof
+                        SdkIntegration\FatalErrorListenerIntegration
+                    ) {
+                        return false;
+                    }
+
+                    // We also remove the default request integration so it can be readded
+                    // after with a Yii3 specific request fetcher. This way we can resolve
+                    // the request from Yii3 instead of constructing it from the global state
+                    return ! ($integration instanceof
+                        SdkIntegration\RequestIntegration);
+                }
+            );
+
+            $integrations[] = new SdkIntegration\RequestIntegration(
+                new YiiRequestFetcher($this->container)
+            );
+        }
+
+        return array_merge($integrations, $userIntegrations);
+    }
+
+    /**
+     * Resolve the integrations from the user configuration with the container.
+     *
+     * @return SdkIntegration\IntegrationInterface[]
+     */
+    private function resolveIntegrationsFromUserConfig(): array
+    {
+        // Default Sentry SDK integrations
+        $integrations = [
+            new Integration(),
+            new Integration\ExceptionContextIntegration(),
```

Comment:
Is it also default? Looks like a custom one.

Reply:
context integration were move to disableble default integration